sourcegraph/sourcegraph
# App: Change system tray icon

Update to the black-and-white version of the tray icon at the correct size (32x32px)

Rendering on light and dark wallpaper:

<img width="269" alt="Screenshot 2023-05-01 at 4 50 38 PM" src="https://user-images.githubusercontent.com/602886/235528940-a851b834-4c59-4db8-a026-ebaede90edde.png">
<img width="269" alt="Screenshot 2023-05-01 at 4 50 29 PM" src="https://user-images.githubusercontent.com/602886/235528956-dec68227-4b2d-4890-b40b-842c0e9755cf.png">

Note that the light and dark versions are rendered depending on whether the wallpaper itself is light or dark, not based on the light or dark UI mode.

The light and dark modes are automatically enabled by the "template icons" feature (`iconAsTemplate: true`)

## Test plan

- Launch app and check icons
- Switch between a light and dark wallpaper to trigger change
